### PR TITLE
fix: fix the dapp-agoric-basics page intro content

### PR DESCRIPTION
**Pull request by ansabgillani**
Fix the intro for `dapp-agoric-basics` page. Originally the page doesn't have intro to the basics repo, rather just an intro to a specific smart contract implemented inside the repo

### CODE REVIEW

File: main/guides/getting-started/tutorial-dapp-agoric-basics.md [hunk 1]
```diff
@@ -1,10 +1,10 @@
 # The `dapp-agoric-basics` Tutorial
 
 ## Introduction
-In this tutorial you will install the `dapp-agoric-basics` dapp. This dapp allows a user to purchase concert tickets for an event. This event has three types of tickets:
-- Tickets near the front are the most expensive
-- Tickets in the middle are priced between expensive and cheap tickets
-- Tickets in the back are the cheapest priced tickets
+In this tutorial you will install the `dapp-agoric-basics` dapp. This dapp is a collection of basic usecases of Agoric smart contract.
```
Comment (Jovonni) [inline main/guides/getting-started/tutorial-dapp-agoric-basics.md:4]: change ```usecases``` to ```use cases```

**Review by Jovonni — CHANGES_REQUESTED** [review]
minor commentary. Lgtm other then that. good stuff

**Comment by amessbee** [comment]
@Jovonni if you approve this minor change, we can merge this right away!

**Comment by amessbee** [comment]
This is not needed since changes are covered in #1166.

**Pull request closed without merging**